app/javascript/lib/highlight.js

```diff
@@ -0,0 +1,18 @@
+// If an element to highlight is declared in the document head, add a CSS class
+// to highlight it. Remove the class when the highlight animation completes
+// so it can be triggered on the same element again if needed.
+export function highlightElement () {
+  const metaTag = document.querySelector("meta[name='highlight-element']")
```

Comment:
I probably just missed it, but what is actually putting this meta tag into the dom?

Reply:
You didn't! I forgot to push one commit. e56808c contains the changes to the admin layout that are required to make both the flash highlighting as well as the morphing refresh work.